packages/web-components/src/components/cbp-app-header/cbp-app-header.tsx

```diff
@@ -7,9 +7,46 @@ import { Component, Host, h, Prop } from '@stencil/core';
 export class CbpAppHeader {
 
   
+  private navItems: HTMLCbpNavItemElement[] = [];
+
+  @Element() host: HTMLElement;
+
   /** Specifies the context of the component as it applies to the visual design and whether it inverts when light/dark mode is toggled. Default behavior is "light-inverts" and does not have to be specified. */
   @Prop({ reflect: true }) context: "light-inverts" | "light-always" | "dark-inverts" | "dark-always";
 
+
+
+  initNavItemset() {
+    // check for a default navItem, otherwise set the first one active
```

Comment:
Unlike tabs, I don't think this is a good assumption - you can't just assume the first nav is active/current. It's on the app to make this work and make sense. Maybe make the story default the first item to current to show the styling and behavior.

Reply:
update var init to not assume selected state